tests/search/search_tests.py

```diff
@@ -488,3 +490,31 @@ def test_adv_search_on_null_file_node(self):
         results = search_results(request=request)
         results = JSONDeserializer().deserialize(results.content)["results"]["hits"]
         self.assertEqual(1, len(results["hits"]))
+
+    def test_no_resources_from_unpublished_graphs(self):
+
+        test_graph = Graph.objects.get(name="Resource Test Model")
+
+        Resource.objects.create(
+            graph=test_graph,
+            name="Resource from unpublished graph",
```

Comment:
```suggestion
            name={"en": "Resource from unpublished graph"},
```

Reply:
fyi the name field on a Resource is json

Reply:
The I18n field is "smart" enough to wrap this as JSON, keying off the current language during the trip to the database. I think since that's supposed to be a usability feature, keeping the tests modeling ordinary usage (as it is here) might be good, but I don't have a strong opinion on what's clearer.